### DISCUSSION TITLE
MQT: tele/tasmota_5273F9/INFO3 = {"Info3":{"RestartReason":{"Exception":29,"Reason":"Exception","EPC"

### DISCUSSION THREAD

**Discussion by DSC4465** [arendst/Tasmota#12082] [Q&A] [answered]
Hello,

I have a problem with a SONOFF TH16 module which is unreachable. I manage to get out of this state by restarting the WIFI terminal to which it is connected. Below are the traces of the restart in the console. Have you ever had this type of problem? do you have an idea ? configuration problem?

Thank you ;)

![image](https://user-images.githubusercontent.com/82834741/117872944-8bcdda00-b29f-11eb-8dfe-1fb968f8811f.png)

**Comment by barbudor** [comment, marked as answer]
You seems to have difficulties to reach AP1 and can't reach AP2 at all.
Is AP2 actually around or is it a backup AP that is not available ?
Is AP1 an extender (as _2GEXT suggest it) ? Sometimes that doesn't work very well.
You may try `reset 3` to clear the Wifi radio parameter (that doesn't clear the Wifi AP and password settings) and then power off then on  `SetOption56 1` may also help Tasmota to rescan network at each restart.
